lib/grover/js/processor.js

```diff
@@ -190,6 +202,23 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       await page.hover(hoverSelector);
     }
 
+    if (errors.length > 0) {
+      function RequestFailedError(errors) {
+        this.name = "RequestFailedError";
+        this.message = errors.map(e => {
+          if (e.failure()) {
+            return e.failure().errorText + " at " + e.url();
+          } else if (e.response() && e.response().status()) {
+            return e.response.status() + " " + e.url();
```

Comment:
and shouldn't this be `e.response().status()` ?
```suggestion
            return e.response().status() + " " + e.url();
```
I think this also deserves a separate spec test. Ie one where you include a URI to a host that DOES exist, but where the image path doesn't (ie get a 404 back and thus test this part of the code). Say http://google.com/foo.png

Reply:
Added some more tests (one success case, one redirect) to expose this.